src/platform/workers/sync/sync-worker.ts

```diff
@@ -19,15 +19,14 @@ import assetPaths from "../../web/sdk/paths/vite";
 
 declare const self;
 
-// Vite makes the paths relative (e.g. ./assets/olm.abc123.js) but we want the absolute path, so we strip the
-// leading `.`.
-// TODO: Fix this at vite level.
```

Comment:
I assume, this was handled earlier or not needed anymore?

Reply:
This was fixed in another way, see `assetPath()` function in same file.